src/IncrementalChecker.ts

```diff
@@ -9,6 +9,7 @@ import WorkSet = require('./WorkSet');
 import NormalizedMessage = require('./NormalizedMessage');
 import CancellationToken = require('./CancellationToken');
 import minimatch = require('minimatch');
+import vueParser = require('vue-parser');
```

Comment:
It would be better to use [vue-template-compiler](https://www.npmjs.com/package/vue-template-compiler) parseComponent function. Return type is SFCDescriptor, see flow typings https://github.com/vuejs/vue/blob/dev/flow/compiler.js#L172-L177